build/Dockerfile

```diff
@@ -84,7 +84,7 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 	&& printf "%s\n" "deb https://pkgs.nginx.com/app-protect/${NGINX_PLUS_VERSION}/debian ${DEBIAN_VERSION} nginx-plus" \
 	"deb https://pkgs.nginx.com/app-protect-security-updates/debian ${DEBIAN_VERSION} nginx-plus" > /etc/apt/sources.list.d/nginx-app-protect.list \
 	&& apt-get update \
-	&& apt-get install --no-install-recommends --no-install-suggests -y app-protect app-protect-attack-signatures app-protect-threat-campaigns \
```

Comment:
Does the compiler include a default set of attack-signatures and threat-campaigns?
Since those would need to be available on the data plane for JSON APPolicy processing.
This is the main reason customers that deploy NIC + NAP WAF update their images daily today.
AFAIK - only WAF bundles includes signatures and campaigns.

Reply:
We should still be installing signatures and campaigns for JSON APPolicy customers. I think these might come included with the app-protect package, but in order to get the latest up-to-date versions on our builds (and as Brian alludes to, currently rebuilding the image is the only way to fetch the latest signatures and campaigns), we need to keep these here in our Dockerfile.

Reply:
`app-protect-compiler`, `app-protect-common` and `app-protect-engine` are already installed by `app-protect` so we don't need to add them here. 
`app-protect-attack-signatures` and `app-protect-threat-campaigns` are dependents of `app-protect` as well and in theory we could remove those too since this layer would be rebuilt anyway, but I think it's better to keep them here for what @ciarams87 said just in case.

Reply:
thank you for providing feedback; updates applied